src/TableRow.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from "react";
+
+import DaysCalculator from "./DaysCalculator";
+
+// var moment = require('moment');
+// moment().format();
+
+
+class TableRow extends Component {
+  constructor(props){
+    super(props)
+  this.state = {
+    selected: false
+  }}
+
+  clickHandle = ()=> {
+    this.setState(currentState => {
+      return{
+      selected: !currentState.selected}})
```

Comment:
Excellent work @solasobowale!! Really well done figuring this problem out - it's one of the harder questions 😄